src/lib.rs

```diff
@@ -483,12 +493,14 @@ impl<L: Language> Formatter<L> {
         let recurse_result = self.convert_impl(rem, items_left - 1);
 
         let word = self.lang.get_word(dtu, x);
+        let between = self.lang.between_value_and_word();
+        let between_chunk = self.lang.between_chunks();
 
         match (self.lang.place_unit_before(x), recurse_result.is_empty()) {
-            (true, true) => format!("{word} {x}"),
-            (true, false) => format!("{word} {x} {recurse_result}"),
-            (false, true) => format!("{x} {word}"),
-            (false, false) => format!("{x} {word} {recurse_result}"),
+            (true, true) => format!("{word}{between}{x}"),
```

Comment:
Are inline format strings supported by the Rust version this crate aims to support?

Reply:
I assume so, because they're used in the version on `master` as well (e.g. in `format!("{word} {x} {recurse_result}")`, all the variables are captured from the local scope like `{between}` is in the changed version).

Thanks for the quick review, I saw the repo wasn't very active so I figured I was just shouting into the void :)

Reply:
`cargo +1.63 test` passed (on my machine at least), so I think we're good!